frontend/src/pages/post/[slug].tsx

```diff
@@ -22,66 +20,61 @@ interface Props {
 
 export default function PostPage(props: Props) {
     const post = props.post;
-    
-    return <div>
-        <HeadArticle post={post}/>
-        <Banner title={post.title}
-                subtitle={<>
-                    {post.author}
-                    <span className="mx-2">&#xB7;</span>
-                    {dateToText(new Date(post.publishedAt))}
-                </>}
+
+    const content =
+        post.type == PostType.PETITION && post.petition ? (
+            <PetitionBody
+                post={post}
+                petition={post.petition}
+                fallbackBanner={props.fallbackBanner}
+            />
+        ) : (
+            <PostBody post={props.post} fallbackBanner={props.fallbackBanner} />
```

Comment:
Verschillen ze echt zo sterk dat je de verschillende logica niet af kan vangen als conditional renders in een PostBody?

Reply:
Is wel waar